homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -713,6 +724,9 @@ async def _wake_word_audio_stream(
                         code="wake-word-timeout", message="Wake word was not detected"
                     )
 
+            if self.abort_wake_word_detection:
+                raise WakeWordDetectionAborted
```

Comment:
This check could be moved before the `yield` instead, I'm not sure it matters.

Reply:
Might as well move it before the yield to abort faster